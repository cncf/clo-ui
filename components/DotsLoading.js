module.exports = require('../dist/cjs/components/DotsLoading.js');
