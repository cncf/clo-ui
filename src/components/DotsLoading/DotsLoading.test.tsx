import { render, screen } from '@testing-library/react';

import { DotsLoading } from './DotsLoading';

describe('DotsLoading', () => {
  afterEach(() => {
    jest.resetAllMocks();
  });

  it('creates snapshot', () => {
    const { asFragment } = render(<DotsLoading />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('renders proper content', () => {
    render(<DotsLoading />);

    expect(screen.getByRole('status')).toBeInTheDocument();
  });
});
