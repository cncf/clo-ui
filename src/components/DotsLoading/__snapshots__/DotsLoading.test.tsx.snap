// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DotsLoading creates snapshot 1`] = `
<DocumentFragment>
  <div
    class="d-flex flex-row undefined"
    role="status"
  >
    <div
      class="dot dot1 dot"
    />
    <div
      class="dot dot2 dot"
    />
    <div
      class="dot dot3 dot"
    />
  </div>
</DocumentFragment>
`;
